src/hooks/ProcessedRichNote/ProcessedRichNote.ts

```diff
@@ -22,9 +29,24 @@ export const useProcessedRichNote = (note: string | undefined) => {
    *   [www.wikipedia.org],
    * ]
    */
+  // Add newline before any links (starts with 'https' or 'ipfs') in the note that don't already have them
+  const formattedNote =
```

Comment:
Now we can render images without using the image uploader - user can paste the link directly into the memo and it will render